### PR TITLE
[Add Email] Created the add email form

### PR DESCRIPTION
**Pull request by fnormand01**
# Description

I created a form that will allow the candidates to add an email response for the specified email. This form is displayed when you click on **'Add email response'** in the inbox.

**Note that the radio buttons are not functional and the **Save response** button has the same behavior than before.**

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

**Add Email Form - Part 1:**

image

**Add Email Form - Part 2:**

image

# Testing

Manual steps to reproduce this functionality:

1.  Start the eMIB Sample Test.
2.  Complete the _pre-test_ process.
3.  Open the _Inbox_ tab.
4.  Click on _Add email response_ and make sure that the form looks good.

# Checklist

Applicable for all code changes.

- [ ] ~~I have commented my code, particularly in hard-to-understand areas~~
- [ ] ~~I have made corresponding changes to the documentation~~
- [x] My changes generate no new compiler warnings
- [ ] ~~My changes generate no new accessibility errors and/or warnings~~
- [ ] ~~I have added tests that prove my fix is effective or that my feature works~~
- [x] I have researched WCAG2.1 accessibility standards and met them in this PR (can be navigated using a keyboard)
- [ ] My changes look good on IE 10+, Firefox, and Chrome

### CODE REVIEW

File: frontend/src/css/cat-theme.css [hunk 1]
```diff
@@ -196,5 +196,5 @@ Progess Steps
   left: 50%;
   transform: translate(-50%, -50%) !important;
   min-width: 500px !important;
-  max-width: 750px !important;
+  max-width: 1000px !important;
```
Comment (caleybrock) [inline frontend/src/css/cat-theme.css]: Is this still required?

File: frontend/src/components/eMIB/EditEmail.jsx [hunk 2]
```diff
@@ -1,10 +1,182 @@
...
+      height: 32,
+      lineHeight: "2.1em"
+    }
+  },
+  response: {
+    textArea: {
+      padding: "6px 12px",
+      border: "1px solid #00565E",
+      borderRadius: 4,
+      width: "99.3%",
```
Comment (caleybrock) [inline frontend/src/components/eMIB/EditEmail.jsx]: 100% for these so we don't have decimal values.

**Comment by caleybrock** [comment]
There seems to be a bug in French, and a little bit of odd spacing with IE. We'll definitely need to fix the French, but IE doesn't look that bad. Is probably okay to use as is.

File: frontend/src/css/cat-theme.css (hunk 1 above)
Comment (fnormand01) [reply frontend/src/css/cat-theme.css]: Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/178/commits/85f076d65301c286f95812f3b434c75b7f6ae2c6

File: frontend/src/components/eMIB/EditEmail.jsx (hunk 2 above)
Comment (fnormand01) [reply frontend/src/components/eMIB/EditEmail.jsx]: Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/178/commits/85f076d65301c286f95812f3b434c75b7f6ae2c6

**Pull request merged**